Cargo.lock

```diff
@@ -460,6 +460,7 @@ dependencies = [
  "gaol 0.0.1 (git+https://github.com/servo/gaol)",
  "gfx 0.0.1",
  "gfx_traits 0.0.1",
+ "hyper 0.10.5 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I recommend removing this change, starting a mach build, and committing the results once "compiling foo" starts appearing.